scripts/install.sh

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env bash
+
+{ # this ensures the entire script is downloaded #
+
+BINARY_DEST=${2:-/usr/local/bin}
+BINARY_NAME="rhoas"
+SRC_ORG="bf3fc6c"
+SRC_REPO="cli"
+OS_TYPE="linux"
+OS_LONG_BIT=""
+API_BASE_URL="https://api.github.com"
+API_RELEASE_URL="${API_BASE_URL}/repos/${SRC_ORG}/${SRC_REPO}/releases/latest"
+DOWNLOAD_DIR="/tmp"
+
+has_in_path() {
+  type "$1" > /dev/null 2>&1
+}
+
+source_file() {
+  # shellcheck disable=SC1090
+  source "$1" 2> /dev/null
+}
+
+# get OS type
+if [[ "$OSTYPE" == "darwin"* ]]; then
```

Comment:
This would easily work on windows when we add extra os type:
https://gist.github.com/bermi/4125858

Reply:
I have windows machine can verify that for you

Reply:
That is when you run this from WSL or Bash on Windows right? In which case, the OS Type would be Linux?

Reply:
Yes. That was the problem. Download linux where is should be windows.

Reply:
Did you try to run it on Bash on Windows? I got `bash: line 1: 404:: command not found` because of line 1, so it did not even make it that far.

Reply:
Yes. It downloaded binary for me.
I was using bash subsystem. Dam windows. 
I can install git bash if you want and try git bash as well

Reply:
No need.

Run `echo $OSTYPE` on bash WSL and windows and the output is "linux-gnu". Therefore it should be Linux?